cpp/src/gandiva/expr_decomposer_test.cc

```diff
@@ -32,7 +32,7 @@ using arrow::int32;
 
 class TestExprDecomposer : public ::testing::Test {
  protected:
-  FunctionRegistry registry_;
+  FunctionRegistry& registry_ = *default_function_registry();
```

Comment:
We prefer either const-refs or non-const pointers, so here:
```suggestion
  FunctionRegistry* registry_ = default_function_registry();
```

Reply:
Got it. I've updated 3 places like this to use non-const pointers.